web/src/components/jobs/RunInfo.tsx

```diff
@@ -30,7 +30,7 @@ const RunInfo: FunctionComponent<RunInfoProps> = props => {
           <Box mb={1}>
             <MqText subheading>{i18next.t('jobs.runinfo_subhead')}</MqText>
           </Box>
-          <MqJson code={run.facets} />
+          <MqJsonView data={run.facets} placeholder='Search' />
         </Box>
       )}
     </Box>
```

Comment:
Can we remove MqJson as part of this as we're moving forward with this JSON viewer implementation.

Reply:
Sure, make sense, it's done